test/test.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+require("dotenv").config();
+const expect = require('expect');
+const app = require('../lib/server.js');
+const request = require('superagent');
+const User = require('../models/User.js');
+let username = 'testingUsername';
+let newUsername = 'newUsername';
+let password = 'testingPassword';
+let token = '';
+
+
+// describe('Create a password for testing the token', done => {
+//     before(done => {
```

Comment:
Never check in commented out code!

GIT has versions, so you should nuke unwanted code and revert to a version that has it if you ever wanted to reuse that.